d3a_interface/sim_results/results_abc.py

```diff
@@ -35,6 +35,13 @@ def update(self, area_result_dict, core_stats, current_market_slot):
         """
         pass
 
+    @abstractmethod
```

Comment:
I would recommend to not make it an abstractmethod, and this will remove the need to override the method in the subclasses. If/when more subclasses need it, we'll convert this method to abstract then.